### PR TITLE
Remove sorting by assignees in Issue Sorter

### LINKED ISSUE #280: Remove sorting by assignees in Issue Sorter
WATcher does not need to sort issues by assignees, since the issue columns are already sorted by assignees. Let's remove this from `issue-sorter.ts` and refactor the tests accordingly too.

### PR DESCRIPTION
**Pull request by MadLamprey**
### Summary:

Fixes #280 

### Changes Made:

- Remove sorting by assignees in `issue-sorter.ts`
- Remove test cases for sorting by assignees in `issue-sorter.spec.ts`

### Proposed Commit Message:

```
Remove sorting by assignees in Issue Sorter

Issue columns are already sorted by assignees in WATcher. 

Let's remove sorting by assignees from the Issue Sorter to reduce redundancy.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "ng": "ng",
     "build:staging": "npm run codegen:gql && ng build --c staging --base-href=https://catcher-org.github.io/WATcher-staging/",
-    "build:prod:web": "npm run codegen:gql && ng build --prod --base-href=https://CATcher-org.github.io/WATcher/",
+    "build:prod:web": "npm run codegen:gql && ng build --prod --base-href=https://catcher-org.github.io/WATcher/",
```
Comment (Arif-Khalid) [inline package.json:8]: As this in unrelated to the current PR, could you create an issue and make this change in a separate PR?

**Review by Arif-Khalid — COMMENTED** [review]
An excellent refactor! Aside from small unrelated change that should be in a separate PR.

**Review by gycgabriel — DISMISSED** [review]
LGTM after fix Arif's comment

**Review by gycgabriel — APPROVED** [review]
Lgtm
